CMakeLists.txt

```
@@ -39,6 +39,7 @@ rapids_cmake_build_type(Release)
 # build options
 option(BUILD_TESTS "Configure CMake to build tests" ON)
 option(BUILD_BENCHMARKS "Configure CMake to build (google) benchmarks" OFF)
+option(RMM_COMPILED "Use spdlog compiled library. Header-only version is used otherwise" OFF)
```

How about we flip the semantics of this? Also a more specific name.

```suggestion
option(RMM_HEADER_ONLY_SPDLOG "Use header-only spdlog. Compiled version is used otherwise." ON)
```

Note you had a different name below anyway.

---

Oops. I kept going back and forth about the name and it looks like I ended up not even getting the option right in the end.